### DISCUSSION TITLE
Unicode selection fail

### DISCUSSION THREAD

**Discussion by kdev** [prisma/prisma#6127] [Q&A] [answered]
So I have a schema that looks something like this:
```prisma
model Emoji {
    id   Int    @id @default(autoincrement())
    name String
}
```
`name` is a column that gets filled with unicode emoji values.
If I now try to query like this:
```js
const emoji = await prisma.emoji.findFirst({
    where: {
        name: emojiUnicode,
    }
});
```
the response includes every item where `name` is a unicode char.

Did anyone experience the same error? Or can someone reproduce it?

I know that a workaround would be to use a raw query but still want to ask if someone could help out.

Thanks in advance!

**Comment by ryands17** [comment]
@KokosnussDEV 👋 

Could you provide an example unicode value so that I can check on my end?

**Reply by kdev** [reply]
I used the unicode values of  🇩🇪  and  🇬🇧

**Reply by ryands17** [reply]
Cannot reproduce your issue. Could you share the Prisma call where you have inserted values and called `findFirst`?

**Reply by kdev** [reply]
So the schema I sent is shortened down a bit, it looks like this:
```prisma
model ReactionMessage {
    id        String     @id
    channelId String
    guildId   String
    reactions Reaction[]
}

model Reaction {
    id   Int    @id @default(autoincrement())
    name String
    role String

    ReactionMessage   ReactionMessage @relation(fields: [reactionMessageId], references: [id])
    reactionMessageId String
}
```
So to insert a "Reaction" I first of all create a "ReactionMessage" which works totally fine.
Then I use `prisma.reaction.count()` to look if the `name`, which is an emoji, does already exist in combination with `reactionMessageId` and `role` which works completely fine as well.
I'm inserting like this:
```js
await prisma.reaction.create({
    data: {
        reactionMessageId,
        name: matches.emoji, // the emoji is extracted from a string using this RegExp `/(?<emoji>\p{Emoji_Presentation}+|<:.*?:\d{18,19}>)/`
        role: role.id,
    },
});
```
The find first call looks like this:
```js
const reactionEntry = await prisma.reaction.findFirst({
    where: {
        AND: {
            name: reaction.emoji.name, // This is, when I log it, a unicode value
            reactionMessageId: reaction.message.id,
        },
    },
});
```

**Reply by kdev** [reply]
@ryands17 are you still on this?

**Reply by ryands17** [reply, marked as answer]
I couldn't reproduce this. I would suggest opening an [issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=bug_report.md&title=) with a reproduction so that we can look into this.

**Reply by kdev** [reply]
Before I create an isssue on this could you share some DB insights like collation and MySQL version you couldn't reproduce this error with?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
